persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/StructuralVariantMyBatisRepositoryTest.java

```diff
@@ -0,0 +1,193 @@
+package org.cbioportal.persistence.mybatis;
+
+import org.cbioportal.model.StructuralVariant;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Configurable;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.List;
+import java.util.ArrayList;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("/testContextDatabase.xml")
+@Configurable
+public class StructuralVariantMyBatisRepositoryTest {
+
+    @Autowired
+    StructuralVariantMyBatisRepository structuralVariantMyBatisRepository;
+    
+    @Test
+    public void fetchStructuralVariantsNoSampleIdentifiers() throws Exception {
+
+        List<String> molecularProfileIds = new ArrayList<String>();
+        List<Integer> entrezGeneIds = new ArrayList<Integer>();
+        List<String> sampleIds = new ArrayList<String>();
+
+        molecularProfileIds.add("study_tcga_pub_sv");
+        entrezGeneIds.add(57670);
+
+        List<StructuralVariant> result = 
+                structuralVariantMyBatisRepository.fetchStructuralVariants(molecularProfileIds, 
+                        entrezGeneIds, sampleIds);
+
+        Assert.assertEquals(2,  result.size());
+        StructuralVariant structuralVariantFirstResult = result.get(0);
+        Assert.assertEquals("study_tcga_pub_sv", structuralVariantFirstResult.getMolecularProfileId());
```

Comment:
Why comparing each attributes one by one? Can't we just assert the whole object?